annotator-core/src/test/java/edu/ucr/cs/riple/core/DownstreamAnalysisTest.java

```diff
@@ -40,8 +40,8 @@ public void publicMethodWithDownstreamDependencyEnabled() {
     coreTestHelper
         .addExpectedReports(
             // Change reduces errors on target by -4, but increases them in downstream dependency
-            // DepA by 3, DepB by 4 and DepC by 2. Hence, the total effect is: 5.
-            new TReport(new OnMethod("Foo.java", "test.target.Foo", "returnNullableBad(int)"), 5),
+            // DepA by 3, DepB by 4 and DepC by 3. Hence, the total effect is: 6.
+            new TReport(new OnMethod("Foo.java", "test.target.Foo", "returnNullableBad(int)"), 6),
```

Comment:
What changed that changed the effects here and why is it part of this PR about `@NullUnmarked` classes? I thought effects were always computed before adding any suppressions/null-unmarked regions... is this related to the other change above I asked about? The `error.isSingleFix()` check? If so, should that fix be part of this PR?

Reply:
Yes the bug fix is in #136  and the base of this PR now changed to branch in #136.